docs/tests/conditionals.rst

```diff
@@ -20,17 +20,16 @@ of ``SKIPPED``:
 .. code:: yaml
 
     basic_test:
+        # This basic test has two keys to resolve under only_if.
```

Comment:
Talk about consequences, not the obvious.
`# Both 'user' and 'weekday' must match for this test to run.`